tests/security/openscap/oscap_generating_fix.pm

```diff
@@ -35,25 +35,17 @@ sub run {
     assert_script_run "oscap xccdf generate fix --template urn:xccdf:fix:script:sh --profile standard --output $fix_script xccdf.xml";
 
     my $script_output = script_output "cat $fix_script";
-    if ($script_output =~ m/securettyecho/) {
```

Comment:
I just feel confuse, if this issue not happened but not affect your test, why not just leave it there so can avoid any regression?

Reply:
Agree with you too :)
The bug was fixed and these code is useless atm.
So delete it is good too.